### PR TITLE
Update to pagination service to scrub .git from Gerrit repo URLs

### PR DESCRIPTION
**Pull request by autumnfound**
Reformatted, only change is in getProjectsInternal

### CODE REVIEW

**Review by mbarbero — CHANGES_REQUESTED** [review]
It misses some unit tests

File: src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
- * Copyright (C) 2020 Eclipse Foundation
- * 
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- * 
- * SPDX-License-Identifier: EPL-2.0
- ******************************************************************************/
+/**
+ * ***************************************************************************** Copyright (C) 2020
```
Comment (chrisguindon) [inline src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java]: Copyright text should be on a new line like the previous version!

File: src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+ * Eclipse Foundation
+ *
+ * <p>This program and the accompanying materials are made available under the terms of the Eclipse
```
Comment (chrisguindon) [inline src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java]: Are these p tags intentional?

Comment (autumnfound) [reply src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java]: p tags are fine in javadocs, they explicitly create paragraphs and tend to output cleaner. Probably put there by the formatter

Comment (mbarbero) [reply src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java]: issue is that this is not javadoc here. It's a copyright header. I'm pretty sure there is a rule in the Eclipse formatter  engine (independently of the formatting style you're using) to exclude such headers from being formatted.

**Review by mbarbero — APPROVED** [review]

**Pull request merged**